internal/namespaces/instance/v1/testdata/test-list-server-simple.stdout.golden

```diff
@@ -1,2 +1,13 @@
-ID                                    NAME                         TYPE    STATE     ZONE      PUBLIC IP       PRIVATE IP  TAGS  IMAGE NAME            MODIFICATION DATE  CREATION DATE    VOLUMES  PROTECTED  SECURITY GROUP NAME     SECURITY GROUP ID                     STATE DETAIL  ARCH    IMAGE ID
-18e19087-4435-420b-abf7-a47c50a77fe7  cli-srv-mystifying-brattain  DEV1-S  archived  fr-par-1  163.172.170.53              []    Ubuntu Bionic Beaver  few seconds ago    few seconds ago  1        false      Default security group  49822f6c-1d32-465c-a017-720691d159b2  -             x86_64  f974feac-abae-4365-b988-8ec7d1cec10d
+ID                                    NAME                                            TYPE    STATE     ZONE      PUBLIC IP       PRIVATE IP     TAGS                                                                                                                                      IMAGE NAME                                   MODIFICATION DATE  CREATION DATE    VOLUMES  PROTECTED  SECURITY GROUP NAME                              SECURITY GROUP ID                     STATE DETAIL    ARCH    IMAGE ID
```

Comment:
I'm not sure this should have been updated

Why was the corresponding cassette updated ?